CHANGELOG.md

```diff
@@ -24,7 +29,6 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
   - Added `product(for:)` for calculating the product of the `Numeric` property for all elements in `Sequence`. [#1168](https://github.com/SwifterSwift/SwifterSwift/pull/1168) by [MartonioJunior](https://github.com/MartonioJunior)
 - **UIView**
   - Added `removeBlur()` method for removing the applied blur effect from the view. [#1159](https://github.com/SwifterSwift/SwifterSwift/pull/1159) by [regi93](https://github.com/regi93)
-  - Added `makeCircle(diameter:)` method to make the view circular. [#1165](https://github.com/SwifterSwift/SwifterSwift/pull/1165) by [happyduck-git](https://github.com/happyduck-git)
```

Comment:
Why was this removed?

Reply:
Likely a mistake when moving over the commits on Xcode, will fix this as soon as I can